tools/SetupFlow/DevHome.SetupFlow/ViewModels/PackageViewModel.cs

```diff
@@ -106,8 +103,7 @@ public partial class PackageViewModel : ObservableObject
 
     public IReadOnlyList<string> AvailableVersions => _package.AvailableVersions;
 
-    // When in setup target flow don't disable installed packaged.
-    public bool IsInstalled => _setupFlowOrchestrator.IsSettingUpATargetMachine ? false : _package.IsInstalled;
```

Comment:
So we have this here because if the user is settings up a target machine e.g a local VM or a cloud Dev Box, we don't we can't check if they have the package installed or not but we'd still like them to be able to select it in the UI. If its installed on their local machine then they won't be able to select the app from the UI to use when going through the flow.

Reply:
This PR removes this constraint also from the local setup machine, so it should continue to work as for a target machine, and match the behavior for the local machine. But now I am thinking maybe we should hide the info message that shows up when you're targeting a remote machine (first PR image above)

Reply:
I'm cool with hiding it or changing the message to say its only for local machines. But this doesn't block us from checking this in. I'll leave it up to you to decide which one you want to go with

Reply:
Oh actually one thing I realized when in the setup target flow is that when a user has an app already installed and they get to the review page the setup button stays greyed out. Below is after the selected VS code (installed already)
![image](https://github.com/microsoft/devhome/assets/105318831/cbf52478-3b55-42b0-bc73-726e58246c92)


but when they select an app that isn't installed they can now see the EULA message and the checkbox which enables the setup button:  Below is IntelliJ IDEA IDE (not installed)

![image](https://github.com/microsoft/devhome/assets/105318831/70000e06-2488-4cb7-9cb4-94f95735c022)

Reply:
Good catch, I updated the `CanInstall` logic to always be true for a target machine.